app/assets/javascripts/users.js

```diff
@@ -0,0 +1,70 @@
+var ready = function () {
+
+    /**
+     * When the send message link on our home page is clicked
+     * send an ajax request to our rails app with the sender_id and
+     * recipient_id
+     */
+
+    $('.start-conversation').click(function (e) {
+        e.preventDefault();
+
+        var sender_id = $(this).data('sid');
+        var recipient_id = $(this).data('rip');
+
+        $.post("/conversations", { sender_id: sender_id, recipient_id: recipient_id }, function (data) {
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Identifier 'sender_id' is not in camel case.<br>Identifier 'recipient_id' is not in camel case.